src/app/clients/ClientsPage.tsx

```diff
@@ -162,9 +160,10 @@ const ClientsPage: React.FC<{}> = () => {
             });
 
         return () => {
-            supabase.removeChannel(subscriptionChannel);
+            void supabase.removeChannel(subscriptionChannel);
         };
-    }, [startPoint, endPoint, primaryFilters, sortState]);
+    }, [startPoint, endPoint, primaryFilters, sortState, fetchAndDisplayClientsData]);
```

Comment:
I don't think you need this change anymore since we don't call fetchAndDisplayClientsData?

Reply:
ah that makes sense!